web/src/components/molecules/MyPluginsPage/MyPluginsTable.tsx

```diff
@@ -124,21 +125,33 @@ const MyPluginsTable: React.FC<Props> = ({
     [plugins],
   );
 
+  const renderEmpty = () => {
+    return (
+      <RenderEmptyWrapper>
+        <RenderEmptyText>
+          {t("Click the  “ New Plugin ” button to upload your first Plugin")}
```

Comment:
```suggestion
          {t('Click the "New Plugin" button to upload your first Plugin')}
```
“” seems to be some special characters.

Reply:
Yes, I copied them from the design but replaced them as suggested.